src/main/webapp/app/iris/overview/base-chatbot/iris-base-chatbot.component.ts

```diff
@@ -139,17 +167,22 @@ export class IrisBaseChatbotComponent implements OnInit, OnDestroy, AfterViewIni
     shouldAnimate = false;
     hasActiveStage = false;
 
+    isChatHistoryOpen = true;
+
     // User preferences
+    user: User | undefined;
     userAccepted: boolean;
     isScrolledToBottom = true;
     rows = 1;
     resendAnimationActive: boolean;
     public ButtonType = ButtonType;
 
     showDeclineButton = input<boolean>(true);
+    isChatHistoryAvailable = input<boolean>(false);
     @Input() fullSize: boolean | undefined;
     @Input() showCloseButton = false;
     @Input() isChatGptWrapper = false;
+    @Input() showChatSessions: boolean = false;
```

Comment:
please use signals here as well. Might be feasible to migrate this component to signals within this PR